### PR TITLE
Extract new strings (2024-05-09)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -3553,7 +3553,7 @@ Private Tab</source>
         <note>On top of the bottom toolbar, there can be a microsurvey prompt, this is the title for the button that appears on the prompt that allows the user to tap on and navigates them to the microsurvey to respond to.</note>
       </trans-unit>
       <trans-unit id="Microsurvey.Prompt.Close.AccessibilityLabel.v127" xml:space="preserve">
-        <source>Close Survey Prompt</source>
+        <source>Close</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3556]: You will need to udpate the string ID as the string changes here (more info about that [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#string-identifiers-ids))

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -3565,7 +3565,7 @@ Private Tab</source>
         <note>After engaging with the microsurvey prompt, the microsurvey pops up as a bottom sheet for the user to answer, this the title of button on the survey that a user can tap on to submit their responses.</note>
       </trans-unit>
       <trans-unit id="Microsurvey.Survey.Close.AccessibilityLabel.v127" xml:space="preserve">
-        <source>Close Survey</source>
+        <source>Close</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3568]: Same comment about updating string ID

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -3581,7 +3581,7 @@ Private Tab</source>
         <note>After engaging with the microsurvey prompt, the microsurvey pops up as a bottom sheet for the user to answer, this is the title for the header on the screen.</note>
       </trans-unit>
       <trans-unit id="Microsurvey.Survey.Options.LikertScaleOption1.v127" xml:space="preserve">
-        <source>Very Satisfied</source>
+        <source>Very satisfied</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3584]: (This is OK as it's "just" a capitalization change and localizers deal with that accordingly, so no need to edit the string ID)

File: templates/firefox-ios.xliff [hunk 4]
```diff
@@ -3597,7 +3597,7 @@ Private Tab</source>
         <note>On the microsurvey, this is the title for one of the options that the user can select to answer the survey.</note>
       </trans-unit>
       <trans-unit id="Microsurvey.Survey.Options.LikertScaleOption5.v127" xml:space="preserve">
-        <source>Very Dissatisfied</source>
+        <source>Very dissatisfied</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3600]: (Same comment as above, no need to update string ID here so we are good)

**Comment by cyndichin** [comment]
> @cyndichin: Just a couple updates to make. One gentle reminder just in case: iOS strings that land on main in the mobile source repo end up in our Quarantine repo (here) twice a week thanks to automation, and throughout the release cycle. So strings landing on main should ideally be final (as much as possible of course, we know things might have to change), since they get translated by localizers as soon as I approve the l10n Quarantine PR. More info [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#land-good-quality-strings) about this. It's just a couple strings in this case, so it's fine :) Thank you!

thanks @Delphine for the details and clarification! I'll go ahead and create a PR to update the key (ID).

**Comment by cyndichin** [comment]
@Delphine I had accidentally pushed some string changes to main in the code base, so not sure how that impacts the localization process. However, the latest changes for strings on main should be the correct one. Let me know if you see any issues!

**Review by Delphine — APPROVED** [review]

**Pull request merged**